src/plugins/links/public/components/dashboard_link/dashboard_link_component.tsx

```diff
@@ -115,6 +115,8 @@ export const DashboardLinkComponent = ({
 
     const params: DashboardLocatorParams = {
       dashboardId: link.destination,
+      // TODO remove ts-expect-error when linksEmbeddable is converted to api
```

Comment:
What is this error exactly? The `linksEmbeddable` should fulfill all of the interfaces required as far as I can tell.

Reply:
I replaced `@ts-expect-error` with a cast